### PR TITLE
feat: Create routes

### PR DESCRIPTION
**Pull request by christianvtia**
Create routes for problem set, problem, feedback, settings, login, and logout.

### CODE REVIEW

File: animorank-client/src/routes/(home)/+page.server.js
Comment (marcusaurelys) [inline animorank-client/src/routes/(home)/+page.server.js:1]: Add a caption for this to indicate that they're in the right route.

File: animorank-client/src/routes/(home)/[problemset]/+page.server.js
Comment (marcusaurelys) [inline animorank-client/src/routes/(home)/[problemset]/+page.server.js:1]: Same for this one.

File: animorank-client/src/routes/(home)/[problemset]/[problem]/+page.server.js
Comment (marcusaurelys) [inline animorank-client/src/routes/(home)/[problemset]/[problem]/+page.server.js:1]: Same for this one.

File: animorank-client/src/routes/login/+page.svelte
Comment (marcusaurelys) [inline animorank-client/src/routes/login/+page.svelte:1]: Just do it for all blank +page.svelte files

**Review by marcusaurelys — CHANGES_REQUESTED** [review]
The routes are fine. Just add indicators to show that the correct route is shown

File: animorank-client/src/routes/(home)/+page.server.js
Comment (christianvtia) [reply animorank-client/src/routes/(home)/+page.server.js:1]: Will do, thanks.

**Review by marcusaurelys — COMMENTED** [review]
Great job! Approved.

Congratulations on your first approved pull request.

**Pull request merged**